cmd/mesh/manifest.go

```diff
@@ -71,6 +73,8 @@ func ManifestCmd(args *rootArgs) *cobra.Command {
 	mc.AddCommand(mmc)
 	mc.AddCommand(mvc)
 
+	mc.AddCommand(version.CobraCommand())
```

Comment:
The manifest versions subcommand should be already implemented by https://github.com/istio/operator/blob/c9c8bb2c59585f738b1c0563cf9f5fd31491fc81/cmd/mesh/manifest-versions.go
flags are added at: https://github.com/istio/operator/blob/c9c8bb2c59585f738b1c0563cf9f5fd31491fc81/cmd/mesh/manifest.go#L54

And this version is more like a general version for `mesh`.

Reply:
Thanks - fixed.